README.md

```diff
@@ -202,6 +207,16 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 ###xSQLServerPowerPlan
 * **Ensure**: (key) An enumerated value that describes if Min and Max memory is configured
 
+<<<<<<< HEAD
+### xSqlAlias
+
+* **SQLServerName**: The name of Alias (e.g. svr01\inst01).
```

Comment:
After thinking about this I agree and have updated this to be Name in my dev branch. This adheres better to the standards of resource design and makes sense in the context.